### PR TITLE
[Mobile app] codebase cleanup

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Removes unused assets
- [x] Replaces packages with factory methods.
- [x] Removes unnecessary json serialisation.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- [x] Run the app. All features should work as expected.

#### Related PRs?
https://github.com/airqo-platform/AirQo-api/pull/1249

### CODE REVIEW

File: mobile/lib/models/air_quality_reading.dart [hunk 1]
```diff
@@ -27,24 +25,32 @@ class AirQualityReading extends HiveObject {
...
+      referenceSite: json["siteDetails"]["_id"] as String,
+      latitude: json["siteDetails"]["approximate_latitude"] as double,
+      longitude: json["siteDetails"]["approximate_longitude"] as double,
+      country: json["siteDetails"]["country"] as String,
+      name: (json["siteDetails"]["search_name"] ?? json["siteDetails"]["name"])
+          as String,
+      location: (json["siteDetails"]["location_name"] ??
+          json["siteDetails"]["description"]) as String,
+      region: json["siteDetails"]["region"] as String,
+      source: json["siteDetails"]["network"] as String,
```
Comment (NoahNsimbe) [inline mobile/lib/models/air_quality_reading.dart:53]: Data serialisation. Using site approximate coordinates, region, id etc

File: mobile/lib/models/favourite_place.dart [hunk 2]
```diff
@@ -53,17 +52,6 @@ class FavouritePlace extends HiveObject {
...
-  factory FavouritePlace.fromPlaceDetails(PlaceDetails placeDetails) {
-    return FavouritePlace(
-      name: placeDetails.name,
-      location: placeDetails.location,
-      referenceSite: placeDetails.siteId,
-      placeId: placeDetails.placeId,
-      latitude: placeDetails.latitude,
-      longitude: placeDetails.longitude,
-    );
-  }
```
Comment (NoahNsimbe) [inline mobile/lib/models/favourite_place.dart:65]: Removing unused code

**Review by Baalmart — APPROVED** [review]
Thanks @NoahNsimbe for this. This looks good to me.

I am just requesting @Mnoble-19 and @BenjaminSsempala to run this PR to ensure that nothing has been broken by this codebase grooming.

I will then happily perform a merge.

Thanks again!

**Review by BenjaminSsempala — APPROVED** [review]
Tested out, Works well

**Pull request merged**